### PR TITLE
added componentRestrictions paramater

### PR DESCRIPTION
**Pull request by pionl**

### CODE REVIEW

**Comment by flevour** [comment]
Hi @pionl , thanks for your contribution! Could you please change it so that `componentRestrictions` are a sub-key of `geocoderOptions`? This is to keep your change in line with the existing logic.

**Comment by pionl** [comment]
Hi @flevour , sure, I've missed that, much better. Fixed. Thanks

File: jquery.addressPicker.js [hunk 1]
```diff
@@ -64,7 +64,8 @@
                 },
                 geocoderOptions: {
                     appendAddressString: '',
-                    regionBias: ''
+                    regionBias: '',
+                	componentRestrictions : {}
```
Comment (flevour) [inline jquery.addressPicker.js]: @pionl last minor nitpick, could you fix indentation here please? :+1:

**Comment by pionl** [comment]
@flevour sure, changed to IDE, better? (the indentation wont appear in PHPStorm and sublime). I've also added .gitignore to prevent any unwanted system files or project based files. Is it ok? 
best regards,

**Comment by flevour** [comment]
I'd prefer to leave the `.gitignore`. That kind ignores are better fitted into a global gitignore (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore or https://robots.thoughtbot.com/global-gitignore)

Thanks :) :roller_coaster: :rocket:

**Comment by pionl** [comment]
Ok, sorry for the troubles.

**Comment by flevour** [comment]
Thanks @pionl for your contribution!

**Pull request merged**